src/api-v1.js

```diff
@@ -740,26 +740,64 @@ api.declare({
   route: '/state/:workerType',
   name: 'state',
   title: 'Get AWS State for a worker type',
-  scopes: [['aws-provisioner:view-worker-type:<workerType>']],
+  output: 'state-response.json#',
```

Comment:
How about adding a new `/state/:workerType/ec2-objects` / `ec2Objects(..)` endpoint for that purpose?  From a naming perspective, "state" is typically some basic summary information, so I feel like `simple-state` is really just a workaround for `state` already being taken.